### ISSUE TITLE
miss CppConfigFrameworkTargets.cmake

### ISSUE THREAD

**Issue by bittyshadow** [djurodrljaca/CppPluginFramework#4]
严重性	代码	说明	项目	文件	行	禁止显示状态
错误		CMake Error at D:/Temp/CppConfigFramework/code/cmake/CppConfigFrameworkConfig.cmake:1 (include):
  include could not find load file:

    D:/Temp/CppConfigFramework/code/cmake/CppConfigFrameworkTargets.cmake		D:/Temp/CppConfigFramework/code/cmake/CppConfigFrameworkConfig.cmake	1

**Comment by djurodrljaca** [comment]
You need an additional dependency: https://github.com/djurodrljaca/CppConfigFramework

**Comment by bittyshadow** [comment]
i always add it , the error is in CppConfigFramework

**Comment by djurodrljaca** [comment]
I see I closed the issue a bit too fast. I am unable to recheck this at the moment, but it should work. It is possible there are some issues with the latest changes on Windows as I haven't tried to make a Windows build in a while, but it does work on Linux.

I will take a look later today to see what the problem is.

**Comment by bittyshadow** [comment]
okay ,thanks ,also i see in the CppPluginFramework miss CppPluginFrameworkTargets.cmake too

**Comment by djurodrljaca** [comment]
It works fine on Windows for me as well. I did commit a few unrelated changes so that both config and plugin framework projects can be build on Windows with MinGW 8 and that all tests pass, but this should have no effect on your problem.

Did you actually deploy the CppConfigFramework dependency before you tried to use it?

This is the procedure that works for me:
* Build CppConfigFramework (I have custom location for deployment so I set the CMAKE_INSTALL_PATH to that location)
* Deploy CppConfigFramework (make install)
* Build CppPluginFramework (I set the CMAKE_INSTALL_PATH to my custom deployment location and I also add it to CMAKE_PREFIX_PATH so that CppPluginFramework can find CppConfigFramework dependency)
* Deploy CppConfigFramework (make install) if needed

And that's it. If you have a different procedure then try to use the one above. Also make sure you have the latest changes from both config and plugin repositories.
